docs/source-pytorch/cli/lightning_cli_advanced_3.rst

```diff
@@ -209,69 +213,15 @@ If the CLI is implemented as :code:`LightningCLI(MyMainModel)` the configuration
         init_args:
           ...
 
-It is also possible to combine :code:`subclass_mode_model=True` and submodules, thereby having two levels of
-:code:`class_path`.
-
-
-Class type defaults
-^^^^^^^^^^^^^^^^^^^
```

Comment:
I couldn't find where this was moved or why it must be removed. Could you guide the reviewer here :)

Reply:
I did think about adding comments to guide, but then I forgot. Sorry!

This section was duplicated by mistake, see [lightning_cli_expert.rst#class-type-defaults](https://github.com/Lightning-AI/lightning/blob/master/docs/source-pytorch/cli/lightning_cli_expert.rst#class-type-defaults)